### PR TITLE
kata-monitor: add some links when generating pages for browsers

### PR DESCRIPTION
**Pull request by liubin**
Add some links to rendered webpages for better user experience,
let users can jump to pages only by clicking links in browsers.

Fixes: #4061

Signed-off-by: bin <bin@hyper.sh>

### CODE REVIEW

**Review by fgiudici — APPROVED** [review]
@liubin , I love this PR!
Found few typos in comments, but I don't see them as mandatory to fix.
Tested the patch, so nice :-)

File: src/runtime/cmd/kata-monitor/main.go [hunk 1]
```diff
@@ -184,11 +193,33 @@ func indexPage(w http.ResponseWriter, r *http.Request) {
...
+		w.Write([]byte(fmt.Sprintf(formatter, endpoint.path, endpoint.desc)))
+	}
+}
+
+func indexPageHTML(w http.ResponseWriter, r *http.Request) {
+
+	w.Write([]byte("<h1>Available HTTP endpoints:</h1>\n"))
+
+	var formattedString string
+	needLinkPathes := []string{"/metrics", "/sandboxes"}
```
[the remark this answers is not in the document]
Comment (liubin) [reply src/runtime/cmd/kata-monitor/main.go]: fixed

File: src/runtime/cmd/kata-monitor/main.go [hunk 2]
```diff
@@ -184,11 +193,33 @@ func indexPage(w http.ResponseWriter, r *http.Request) {
...
+
+	w.Write([]byte("<ul>"))
+	for _, endpoint := range endpoints {
+		formattedString = fmt.Sprintf("<b>%s</b>: %s\n", endpoint.path, endpoint.desc)
+		for _, linkPath := range needLinkPathes {
```
[the remark this answers is not in the document]
Comment (liubin) [reply src/runtime/cmd/kata-monitor/main.go]: fixed

**Review by lifupan — APPROVED** [review]

**Pull request merged**